src/tools/illink/test/Mono.Linker.Tests.Cases/Reflection/ObjectGetType.cs

```diff
@@ -1616,7 +1616,7 @@ class AnnotatedInterface
 				[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.All)]
 				interface IBase
 				{
-					[Kept]
+					[Kept (By = Tool.Trimmer /* interface method is not target of reflection */)]
```

Comment:
Ah, so looking at this, this is a bug. Good catch. I thing I got confused by the RequirePublicMethods below, thinking the annotation was just PublicMethods (made the same mistake right now as I was trying to repro).

This should be kept because the actual annotation is .All, not just PublicMethods. We have an existing bug for this so I linked it here.